id/index.html

```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Accessibility Myths</title>
+    <link rel="stylesheet" href="./css/styles.css" />
```

Comment:
@franzaurus please update `href` to `/css/styles.css` so styles are displayed properly.